src/stackedbar-chart.js

```diff
@@ -6,6 +6,17 @@ import AbstractChart from "./abstract-chart";
 const barWidth = 32;
 
 class StackedBarChart extends AbstractChart {
+  getBarPercentage = () => {
+    const { barPercentage = 1 } = this.props.chartConfig;
+    return barPercentage;
+  };
+
+  getBarRadius = (ret, x) => {
+    return this.props.chartConfig.barRadius && ret.length === x.length - 1
```

Comment:
Is there a reason why we are only applying the barRadius to only the top bar of the StackedBarChart?